docs/howto/openstack/nova/resize-server.md

```diff
@@ -0,0 +1,125 @@
+# Resizing a server
+
+This guide will walk you through the required steps to change the number of CPU cores and the amount of memory your server has access to, this is done my changing the server's [flavor](/reference/flavors/).
+
+[Resize](https://docs.openstack.org/nova/latest/admin/configuration/resize.html) (or Server resize) is the ability to change the flavor of a server, thus allowing it to upscale or downscale according to user needs. A resize operation is a two-step process for the user: the first step is to perform the resize, and the second step is to either confirm (verify) success and release the old server or to declare a revert to release the new server and restart the old one.
```

Comment:
```markdown
A resize operation is a two-step process for the user:
1. initiate the resize,
2. either confirm (verify) success and release the old server, or
   declare a revert to release the new server and restart the old one.
```

Reply:
Please mark conversations as "resolved" only under the following circumstances:

1. The issue described is fixed in the topic branch (not just locally in your checkout).
2. The issue doesn't _need_ fixing.

I don't see either of these conditions applying here, so I've unresolved the conversation.

Reply:
I understand.

Reply:
Is the proper etiquette that the reviewer marks as resolved?

Reply:
Not necessarily; it's perfectly okay for you to mark it as resolved — after you've pushed the fix. :)